observermedia/eight-day-week
# Fix/121 PHP Version Check

### Description of the Change

Added a minimum PHP version check for 7.4.

Closes #121

### How to test the Change

1. Spin up the site with a version of PHP below 7.4
2. Verify an admin notice has been added alerting the user of the failed check
3. Verify the admin nav doesn't include any of the custom post type options or settings

### Changelog Entry

> Added - >=PHP 7.4 version check

### Credits

Props @bmarshall511 


### Checklist:
